### PR TITLE
Fix input_t.prm in the nonlinear channel flow benchmark

### PR DESCRIPTION
**Pull request by YiminJin**
This PR is a follow-up of the latest issue displayed in #6471 . The template input file `input_t.prm` in `benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow` uses `include input_v.prm` for simplicity, but it makes the bash script `run.sh` produce the same result for different cases, because commands like `-e "s/set Output directory .*/set Output directory = results\/$dirname_clean/g"` does not work. This PR replaces `include input_v.prm` by the actual parameter settings, and the problem is mended. The results with traction boundary conditions are show below:
figure_t

### CODE REVIEW

File: benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm [hunk 1]
```diff
@@ -1,17 +1,132 @@
-# Like input_v.prm and based on the nonlinear channel
-# flow benchmark. This is used to test the traction boundary conditions
+# Like the input_v.prm test and based on the nonlinear channel
+# flow benchmark. This is used to test the velocity boundary conditions
```
Comment (alarshi) [inline benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm]: I think you accidentally changed it. Please change it back to traction.

**Review by alarshi — COMMENTED** [review]
Thank you for noticing this bug and fixing it! 
What you did is correct but I was wondering if we can achieve the same changes by just copying the `Solver parameters` subsection (ln 19- ln 35) and including the `input_v.prm` file as before, since everything else is not changed by the `run.sh` script. 
Besides one small change in the comment, it looks good to me if the tester is happy.

File: benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm [hunk 2]
```diff
@@ -1,17 +1,132 @@
...
   # pressure bc: Prescribe a zero vertical velocity component on the vertical boundaries
+  set Zero velocity boundary indicators       = left, right
   set Prescribed velocity boundary indicators = bottom x: function, top x: function
+
+  subsection Function
+    set Function constants = n = 3
+    set Variable names = x,y
+
+    # For velocity boundary conditions both are used, for pressure boundary conditions only the first (x) component
+    set Function expression = 0;(1e-37/(n+1))*((1e9/8e3)^n)*(((5e3)^(n+1))-(abs(x-(5e3))^(n+1)));
```
Comment (MFraters) [inline benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm]: I think the second part can be zero as well now as it is no y component. It should not matter, but I think it looks more clean than a complicated function.

**Review by alarshi — APPROVED** [review]
Looks good, thank you!
